src/ensembl/src/shared/round-button/RoundButton.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { ReactNode } from 'react';
+import classNames from 'classnames';
+
+import defaultStyles from './RoundButton.scss';
+
+export enum RoundButtonStatus {
+  ACTIVE = 'active',
+  INACTIVE = 'inactive',
+  DISABLED = 'disabled',
+  DEFAULT = 'default'
+}
+
+type Props = {
+  onClick: () => void;
+  buttonStatus: RoundButtonStatus;
```

Comment:
just `status`; since the component itself is called a button, it is obvious that it will be the status of this button

Reply:
That's done.

Reply:
That's why we allow the type string. We can use '0' to display 0 and also ' ' to display an empty badge.